src/org/apache/commons/httpclient/HttpMethodBase.java

```diff
@@ -1341,13 +1348,12 @@ protected void addCookieRequestHeader(HttpState state, HttpConnection conn)
         if ((cookies != null) && (cookies.length > 0)) {
             if (getParams().isParameterTrue(HttpMethodParams.SINGLE_COOKIE_HEADER)) {
                 // In strict mode put all cookies on the same header
-                String s = matcher.formatCookies(cookies);
-                getRequestHeaderGroup().addHeader(new Header("Cookie", s, true));
+            	putAllCookiesInASingleHeader(host, matcher, cookies);
```

Comment:
Consider using spaces (instead of tabs) for indentation, to keep the same format as the rest of the file.